README.md

```diff
@@ -45,10 +45,7 @@ Also, you can read more about how we use Go at AppOptics in our [blog post annou
 
 To get tracing, you'll need a [(free) AppOptics account](http://www.appoptics.com).
 
-In the product install flow, **choose to skip the webserver and select any language.**  (You'll notice the absence of Go because we're currently in beta.)
-
-Follow the instructions during signup to install the Host Agent (“tracelyzer”). This will also
-install the liboboe and liboboe-dev packages on your platform.
+Follow the instructions during signup to install the Go Agent.
```

Comment:
I think it's worth mentioning that the service key is required and then link to the Configuration section.